proto/verifiable-credential/verifiable-credential.proto

```diff
@@ -41,24 +49,102 @@ definition: https://w3c.github.io/vc-data-model/
 }
 */
 
+// VerifiableCredential represents a verifiable credential
 message VerifiableCredential {
-  repeated string context = 1;
+  option (gogoproto.equal) = true;
+  option (gogoproto.goproto_getters)  = false;
+
+  // @context is spec for verifiable credential.
+  repeated string context                          = 1 [(gogoproto.jsontag) = "@context,omitempty"];
+
+  // The value of the id property MUST be a single URI. It is RECOMMENDED
+  // that the URI in the id be one which, if dereferenced, results in a
+  // document containing machine-readable information about the id.
   string id = 2;
+
+  // The value of the type property MUST be, or map to (through interpretation
+  // of the @context property), one or more URIs. If more than one URI is
+  // provided, the URIs MUST be interpreted as an unordered set.
   repeated string type = 3;
+
+  // The value of the issuer property MUST be either a URI or an object
+  // containing an id property. It is RECOMMENDED that the URI in the issuer
+  // or its id be one which, if dereferenced, results in a document containing
+  // machine-readable information about the issuer that can be used to verify
+  // the information expressed in the credential.
   string issuer = 4;
+
+  // A credential MUST have an issuanceDate property. The value of the issuanceDate
+  // property MUST be a string value of an [RFC3339] combined date and time string
+  // representing the date and time the credential becomes valid, which could
+  // be a date and time in the future. Note that this value represents the earliest
+  // point in time at which the information associated with the credentialSubject
+  // property becomes valid.
   string issuance_date = 5;
```

Comment:
I think the RFC3339 format is only required when serializing to json-ld, but we can use a date as model.
```suggestion
  google.protobuf.Timestamp issuance_date = 5 [(gogoproto.stdtime) = true];
```